use_map: False  # True
grasp_check: False
safety_distance: 0.08

## find_grasppoint_server parameters
grasppoint_offset_haf: 0.04 #0.07
use_table_grasp_checker: True

objects_keys:
  "1": "002_master_chef_can"
  "2": "003_cracker_box"
  "3": "005_tomato_soup_can"
  "4": "006_mustard_bottle"
  "5": "009_gelatin_box"
  "6": "010_potted_meat_can"
  "7": '011_banana'
  "8": '021_bleach_cleanser'
  "9": '024_bowl'
  "10": '025_mug'
# 11: '035_power_drill'
  "11": '061_foam_brick'
  "12": '000002'
  "13": '000003'


objects_display_names:
  "1": "Coffee Master Chef"
  "2": "Cracker Cheezit"
  "3": "Tomato Soup"
  "4": "Mustard"
  "5": "Gelatin Jello"
  "6": "Potted Meat Spam"
  "7": 'Banana'
  "8": 'Bleach Cleanser Soft Scrub'
  "9": 'Bowl'
  "10": 'Mug'
# 11: 'Power Drill'
  "11": 'Foam Brick'
  "12": 'CheezeIt'
  "13": 'i dont know yet'


placement_grasp: 0
placement_object_dimensions: [0.1, 0.1, 0.1] # -x, +x, +z
placement_use_config_distance: True
placement_object_name: name of object
placement_resolution: 0.1 #0.1
placement_steps: 0.05
placement_sort: 0
allow_drive: True
allow_placement: True
point_cloud_topic: '/hsrb/head_rgbd_sensor/depth_registered/rectified_points'
global_frame: 'map'
camera_frame: 'head_rgbd_sensor_rgb_frame'
use_mesh: False